internal/scheduler/scheduler_metrics.go

```diff
@@ -0,0 +1,161 @@
+package scheduler
+
+import (
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/armadaproject/armada/internal/scheduler/interfaces"
+)
+
+const (
+	NAMESPACE = "armada"
+	SUBSYSTEM = "scheduler"
+)
+
+type SchedulerMetrics struct {
+	// Cycle time when scheduling, as leader.
+	scheduleCycleTime prometheus.Histogram
+	// Cycle time when reconciling, as leader or follower.
+	reconcileCycleTime prometheus.Histogram
+	// Number of jobs scheduled per queue.
+	scheduledJobs prometheus.HistogramVec
+	// Number of jobs preempted per queue.
+	preemptedJobs prometheus.HistogramVec
+}
+
+var (
+	schedulerMetricsPointer *SchedulerMetrics
```

Comment:
```suggestion
	schedulerMetrics *SchedulerMetrics
```
No need to add the type to the variable name imo.